src/homotopies.jl

```diff
@@ -58,14 +59,15 @@ end
 """
     dt!(u, H::AbstractHomotopy, x, t, cache::AbstractHomotopyCache)
 
-Evaluate the homotopy `H` at `(x, t)` and store the result in `u`.
+Evaluate the derivative of the homotopy `H` w.r.t. `t` at `(x, t)` and store the result in `u`.
 """
 function dt! end
 
 """
+    dt(H::AbstractHomotopy, x::AbstractVector
     dt(H::AbstractHomotopy, x::AbstractVector, cache::AbstractHomotopyCache)
 
-Evaluate the homotopy `H` at `(x, t)`.
+Evaluate the derivative of the homotopy `H` w.r.t. `t` at `(x, t)`.
```

Comment:
Maybe one should rather clarify that this is the partial derivative w.r.t. 

1. "the second/last argument of `H`", or
2. "the `t` variables"?

Reply:
I was confused as to whether this is the derivative of

```math
t \mapsto H(x(t), t) : [0,1] \to \C^m
```

which would have been 0 (Davidenko equation).

Reply:
This is the partial derivative of `H: \C^n \times \C \to C\^m` w.r.t. the last argument (which we commonly denote by `t`).

Reply:
Yeah, I know. Do you think it should be clarified in the docstring and, if so, how would you phrase it?

Reply:
I think your current suggestion is fine :)